docs/extensions/guides/renderer-extension.md

```diff
@@ -96,7 +98,7 @@ The above example shows how to create a cluster page, but not how to make that p
 
 ### `clusterPageMenus`
 
-`clusterPageMenus` allows you to add cluster page menu items to the left side menu.
+`clusterPageMenus` allows you to add cluster page menu items to the secondary left nav.
```

Comment:
Agreed. I will work to get this updated elsewhere in the docs. On this page, I think it would be helpful to include screenshots for the first time each appears: in `clusterPageMenus` for "secondary left nav" and in `globalPageMenus` for "left nav." Do you have screenshots for each of these?